admin/taxonomy/class-taxonomy-columns.php

```diff
@@ -168,4 +172,46 @@ private function get_focus_keyword( $term ) {
 		return $term->name;
 	}
 
+	/**
+	 * Checks if a taxonomy is being added via a POST method. If not, it defaults to a GET request.
+	 *
+	 * @return int
+	 */
+	private function get_taxonomy_input_type() {
+		$request_type = filter_input( INPUT_SERVER, 'REQUEST_METHOD' );
+
+		if ( $request_type === 'POST' ) {
+			return INPUT_POST;
+		}
+
+		return INPUT_GET;
+	}
+
+	/**
+	 * Test whether the metabox should be hidden either by choice of the admin
+	 *
+	 * @since 3.1
+	 *
+	 * @param  string $taxonomy (optional) The post type to test, defaults to the current post post_type.
+	 *
+	 * @return  bool        Whether or not the meta box (and associated columns etc) should be hidden
+	 */
+	private function is_metabox_hidden( $taxonomy = null ) {
+		$get_taxonomy_type = filter_input( $this->get_taxonomy_input_type(), 'taxonomy' );
+
+		if ( ! isset( $taxonomy ) && $get_taxonomy_type ) {
+			$taxonomy = sanitize_text_field( $get_taxonomy_type );
+		}
+
+		if ( isset( $taxonomy ) ) {
+			// Don't make static as taxonomies may still be added during the run.
+			$custom_taxonomies = get_taxonomies( array( 'public' => true ), 'names' );
+			$options           = get_option( 'wpseo_titles' );
+
+			return ( ( isset( $options[ 'hideeditbox-tax-' . $taxonomy ] ) && $options[ 'hideeditbox-tax-' . $taxonomy ] === true ) || in_array( $taxonomy, $custom_taxonomies ) === false );
```

Comment:
Can we make this `if` more readable? For example by putting `'hideeditbox-tax-' . $taxonomy` in a `$option_key` variable.